extensions/amp-subscriptions-google/amp-subscriptions-google.md

```diff
@@ -0,0 +1,85 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-subscriptions-google"></a> `amp-subscriptions-google`
+
+<table>
+  <tr>
+    <td class="col-fourty"><strong>Description</strong></td>
+    <td>Dynamically injects ads into an AMP page by
```

Comment:
Dynamically injects ads??? 
I guess this one is a WIP or something?

Reply:
Fixed.